crates/fuel-core/src/service/adapters/p2p.rs

```diff
@@ -49,3 +54,20 @@ impl BlockHeightImporter for BlockImporterAdapter {
         )
     }
 }
+
+impl TxPool for TxPoolAdapter {
+    fn get_all_tx_ids(&self) -> Vec<fuel_core_txpool::types::TxId> {
```

Comment:
nit: can we have a top level import for TxId please?
```suggestion
    fn get_all_tx_ids(&self) -> Vec<TxId> {
```

Reply:
Done